frontend/components/Snowfall/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Snowflake from './components/Snowflake';
+import { canvas } from './styles';
+
+/**
+ * @returns {JSX}
+ */
+const Snowfall = () => (
```

Comment:
snowflakes appear sometimes in the middle of the screen and fall from there. 
for me they should always appear at top and fall down from there

also the design looks to me like the snowfall should only be an overlay for the calendar itself and not the complete page